src/public/inventory.ts

```diff
@@ -133,5 +133,32 @@ export const inventoryPublicHandlers: PublicNatsHandler[] = [
         {
             queue: SERVICE_NAME
         }
+    ],
+    [
+        "PUT",
```

Comment:
there's already a `PUT` on `inventory.*` above, which means that when doing a PUT request on `item-store/inventory/id` both endpoints will be called and one will fail.
We can't have 1 PUT for updating and one PUT for transferring an item. Any suggestion, is there another VERB we could use? Should we use a different URL?